operator/pkg/manifest/installer.go

```diff
@@ -426,26 +427,23 @@ func ApplyManifest(componentName name.ComponentName, manifestStr, version, revis
 	// We sort them by namespace so that we can pass the `-n` to the apply command. This is required for prune to work
 	// See https://github.com/kubernetes/kubernetes/issues/87756 for details
 	namespaces, nonNsCrdObjectsByNamespace := splitByNamespace(objectsNotInLists(objects, nsObjects, crdObjects))
-	var applyErr error
```

Comment:
We seem to be using `applyErr` for aggregating errors. Since this project already imports `*multierror.Error`, it's only fair that we use when the situation calls for it.